src/contract.js

```diff
@@ -4,11 +4,15 @@ const _ = require('lodash');
 // Internal Imports
 const Formatter = require('./formatter');
 const Utils = require('./utils.js');
+const Encoder = require('./encoder');
 
 const SEND_AMOUNT = 0;
 const SEND_GASLIMIT = 250000;
 const SEND_GASPRICE = 0.0000004;
 
+const MAX_BYTES_PER_ARRAY_SLOT = 64;
```

Comment:
this is bodhi-specific constants, leave it as it is now, should move it to `bodhi-core.js` later

Reply:
Ya I will make this more generic later. But actually 64 is always the padded length for any qtum-cli param. Think the naming is a little off though.